src/raspgierry_pi.py

```diff
@@ -0,0 +1,35 @@
+import pygame, sys
+from pygame.locals import *
+from controls.joystick import *
+
+class RaspgierryPi:
+    def __init__(self):
+        pygame.init()
+        self._display = {'width': 320, 'height': 480}
+        native_screen = pygame.display.Info()
+        if native_screen.current_w < self._display['width'] or native_screen.current_h < self._display['height']:
+            # TODO sk: Add error screen for minimum display when available
+            sys.exit(1)
+        self._display_ratio = native_screen.current_w /  self._display['width']
```

Comment:
won't you need display ratio for height as well?

Reply:
Actually not. Default aspect ratio is 4:3. If we'll get e.g. twice bigger resolution then it's enough to count just one dimension for the ratio. If the aspect ratio will be different then one dimension is also enough because we don't want to stretch more in one of the dimensions. In such case, we would make all the textures to look ugly.